backend/app/store/search/search_test.go

```diff
@@ -0,0 +1,281 @@
+package search
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/go-pkgz/syncs"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.etcd.io/bbolt"
+
+	"github.com/umputun/remark42/backend/app/store"
+	"github.com/umputun/remark42/backend/app/store/engine"
+)
+
+func createTestServiceInDir(t *testing.T, sites []string, idxPath string, keepIdx bool) (searcher *Service, teardown func()) {
+	searcher, err := NewService(sites, ServiceParams{
+		IndexPath: idxPath,
+		Analyzer:  "standard",
+	})
+
+	require.NoError(t, err)
+
+	teardown = func() {
+		err := searcher.Close()
+		require.NoError(t, err)
+		if !keepIdx {
+			_ = os.RemoveAll(idxPath)
+		}
+	}
+	return searcher, teardown
+}
+
+func createTestService(t *testing.T, sites []string) (searcher *Service, teardown func()) {
+	idxPath := os.TempDir() + "/search-remark42"
+	_ = os.RemoveAll(idxPath)
+	return createTestServiceInDir(t, sites, idxPath, false)
+}
+
+func TestSearch_Site(t *testing.T) {
```

Comment:
`skip` seems to be missing

Reply:
Added https://github.com/umputun/remark42/pull/757/commits/3e4515b0cbecc0927cfc3a47defbee1f3dd3675e#diff-f933ab083e2ed57c319100cfb2bc25f3fc8493387c6b2fd5531600d3150870aaR96